src/extension/devtools/devtools.ts

```diff
@@ -164,11 +144,6 @@ async function createDevtoolsPanel() {
       connectedToPanel = true;
     }
 
-    if (devtoolsMachine.state.value === "initialized") {
```

Comment:
This is no longer needed since the rest of the messages are now robust enough to not have to "retry" connecting when the panel is first shown 🎉